RELEASE-CHECKLIST.md

```diff
@@ -0,0 +1,46 @@
+# Release Checklist
+
+### 1. Update version and CHANGELOG.md
```

Comment:
Add to the end 'on docker/swarm'. It's clear to us but not necessary to external maintainer.